sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/ServiceBusManagementSerializerTest.java

```diff
@@ -458,29 +458,29 @@ void deserializeSubscriptionDescriptionFeed() throws IOException {
             subscription1, subscription2, subscription3);
 
         final SubscriptionDescriptionEntry entry1 = new SubscriptionDescriptionEntry()
-            .setId("https://sb-java-conniey-5.servicebus.windows.net/topic/Subscriptions/subscription-0?api-version=2017-04")
+            .setId("https://sb-java-conniey-5.servicebus.windows.net/topic/Subscriptions/subscription-0?api-version=2021-05")
             .setTitle(getResponseTitle(subscriptionName1))
             .setPublished(OffsetDateTime.parse("2020-06-22T23:47:53Z"))
             .setUpdated(OffsetDateTime.parse("2020-06-23T23:47:53Z"))
-            .setLink(new ResponseLink().setRel("self").setHref("Subscriptions/subscription-0?api-version=2017-04"))
+            .setLink(new ResponseLink().setRel("self").setHref("Subscriptions/subscription-0?api-version=2021-05"))
             .setContent(new SubscriptionDescriptionEntryContent()
                 .setType("application/xml")
                 .setSubscriptionDescription(subscription1));
         final SubscriptionDescriptionEntry entry2 = new SubscriptionDescriptionEntry()
-            .setId("https://sb-java-conniey-5.servicebus.windows.net/topic/Subscriptions/subscription-session-0?api-version=2017-04")
+            .setId("https://sb-java-conniey-5.servicebus.windows.net/topic/Subscriptions/subscription-session-0?api-version=2021-05")
             .setTitle(getResponseTitle(subscriptionName2))
             .setPublished(OffsetDateTime.parse("2020-06-22T23:47:53Z"))
             .setUpdated(OffsetDateTime.parse("2020-05-22T23:47:53Z"))
-            .setLink(new ResponseLink().setRel("self").setHref("Subscriptions/subscription-session-0?api-version=2017-04"))
+            .setLink(new ResponseLink().setRel("self").setHref("Subscriptions/subscription-session-0?api-version=2021-05"))
             .setContent(new SubscriptionDescriptionEntryContent()
                 .setType("application/xml")
                 .setSubscriptionDescription(subscription2));
         final SubscriptionDescriptionEntry entry3 = new SubscriptionDescriptionEntry()
-            .setId("https://sb-java-conniey-5.servicebus.windows.net/topic/Subscriptions/subscription-session-1?api-version=2017-04")
+            .setId("https://sb-java-conniey-5.servicebus.windows.net/topic/Subscriptions/subscription-session-1?api-version=2021-05")
```

Comment:
@conniey should these services be moved to a team-wide resource (not something with coniey)?